integrationTests/cypress/support/commands/misc_commands.js

```diff
@@ -0,0 +1,109 @@
+// ***********************************************
+// commands for the login/logout flow
+// ***********************************************
+let doguName = "redmine"
+
+/**
+ * Log the test user in the cas.
```

Comment:
This command does not use a fixture (at least not here) so it does not force to log in a "testuser". If this command is used only to log in a specific testuser I would suggest to use the fixture here (like  loginAdmin()) if not just a little change to the method description might be appropriate.

Reply:
I changed the description as the method is designed to log in any user given by the parameters